src/service/roomSocket.js

```diff
@@ -20,14 +20,14 @@ export const createRoom = (
   socket.emit(
     'CREATE_ROOM',
     {
-      max_teams: maxTeams,
-      max_perTeam: maxPerTeam,
-      max_perRoom: maxPerRoom,
+      maxTeams: maxTeams,
```

Comment:
Club it together as `maxTeams` instead of `maxTeams: maxTeams` and repeat it for the rest.